addons/osio-addon/src/main/java/io/fabric8/launcher/osio/client/OsioWitClient.java

```diff
@@ -37,16 +37,41 @@
 @RequestScoped
 public class OsioWitClient {
 
+    protected static final String ERROR_HTTP_CLIENT_MUST_BE_SPECIFIED = "httpClient must be specified"; //$NON-NLS-1$
```

Comment:
`protected` is only used if the class is intended to be extended (which isn't the case here)

Reply:
Sorry, correct, this can be made package protected. Will update!